index.html

```diff
@@ -1799,6 +1824,14 @@ <h3 id="applying">
           next-most-appropriate icon as determined by examining the <a>image
           object's</a> members.
         </p>
+        <p class="note">
+          In order to use <a>adaptive icons</a>, you define the <code>foreground</code>
+          and <code>background</code> properties of <a>image objects</a>. These
+          are special properties only intented for icons and are thus ignored for
+          the <a><code>screenshots</code> member</a>.
+          A <code>src</code> can be added in order as a fallback for vendors which
```

Comment:
Also, make sure the `<a>src</a>` and worst case we fix those later if the link breaks.